packages/cli/src/tools/copyFiles.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import path from 'path';
+import copyAndReplace from './copyAndReplace';
+import walk from './walk';
+
+/**
+ * Copy files (binary included) recursively.
+ */
+function copyFiles(srcPath: string, destPath: string) {
```

Comment:
Created a new utility to just copy files recursively, without replacing stuff around (which could theoretically break some 3rd templates in the future). It's based on `copyAndReplace` but with empty replacements, so binaries are still copied and I verified that it works with `run-android` (previously was breaking)